### ISSUE TITLE
[Feature]: Label / Tag Categories source (Addon/plugin)

### ISSUE THREAD

**Issue by Spennig** [Cidan/BetterBags#645]
### Describe the feature

It would be nice to know which categories are created by which plugin, vs the main addon. This would let me know which to uncheck from the Configure Category list when there seems to be a conflict. 

### Describe the implementation

When I open the Configure Categories list it would be nice to have the source of the list (plugin, main addon, etc) show in the mouseover tooltip. 

It would also be nice to see somehow in the main options, but I've been using the Config off the bags more lately.

![image](https://github.com/user-attachments/assets/01d6bf68-0a7b-40c3-aba5-148b89d08a1a)

**Comment by zeptognome** [comment]
I agree that tracking back categories to plugins would be a good thing, but this is trickier than it appears at first as no one 'owns' a category.

There is a 'note' field that is defined, but not presented to the user yet, but I think something more akin to a table of plugins that have touched a category would work better.

**Comment by grumpypbear** [comment]
I'd like to 2nd this! I have the same issue. I've uninstalled most plugins and started making my own categories to try and work around it, but one of the few I've kept is Teleporters which has resulted in issues like the attached image. Occasionally it'll pop back into the HS category made by BB even if I've turned it off.

The warband item from legion ToS is also now junk when it wasn't previously and I have no idea if that's a plugin or default category issue.

![WoWScrnShot_091224_214231-hearthstone](https://github.com/user-attachments/assets/e43b05c1-3b4c-4eb3-9e90-b936b323281d)
